cobalt/shell/BUILD.gn

```diff
@@ -348,6 +366,74 @@ group("content_shell_app_deps") {
   }
 }
 
+repack("pak") {
```

Comment:
Is this supposed to be just the resources we use or copied as-is from content/shell? If the latter, how would we track that we're only including what we use?